Add Agora Chat 1.0.9 Release Notes (#2672)
AgoraIO/agora_doc_source: en-US/markdown/agora-chat/Understand/agora_chat_release_notes_web.md

This page provides release notes for the Agora Chat Web SDK.

## v1.0.8

v1.0.8 was released on December 19.

#### Improvements

Optimized the callback for a message sending failure to make sure that it is triggered immediately when the network is disconnected.

#### Issues fixed

> The `file_length` parameter in the `create` method does not work when this method is called to create an attachment message.

Reviewer: ```suggestion
The `file_length` parameter in the `create` method did not work when this method was called to create an attachment message.
```
